### PR TITLE
Node Groups Updates - see commits for details

### PR DESCRIPTION
**Pull request by hannamocha**
that don't belong to a given technology

### CODE REVIEW

File: calliope_app/template/models.py [hunk 1]
```diff
@@ -188,6 +187,15 @@ class Meta:
     def __str__(self):
         return '%s' % (self.template_type_variable)
 
+@receiver(pre_save, sender=Template_Type_Loc_Tech_Param)
+def validate_before_save(sender, instance, **kwargs):
+    if sender == Template_Type_Loc_Tech_Param:
+        abstractTechParam = Abstract_Tech_Param.objects.filter(abstract_tech=instance.template_loc_tech.template_tech.abstract_tech, parameter=instance.parameter)
+        if abstractTechParam is not None:
+            print ("Valid parameter for the given technology assoiacted with this Template_Type_Loc_Tech_Param")
```
Comment (jgu2) [inline calliope_app/template/models.py:195]: Use `logger.info` to replace `print`.

**Review by jgu2 — COMMENTED** [review]
comment a minor thing

**Comment by jgu2** [comment]
Resolve this conflict as well.

File: calliope_app/template/models.py (hunk 1 above)
Comment (hannamocha) [reply calliope_app/template/models.py]: Updated!

**Review by jgu2 — APPROVED** [review]

**Pull request merged**